LEAF_Request_Portal/templates/view_session_expire.tpl

```diff
@@ -13,7 +13,7 @@
 <body>
     <div id="header">
         <div style="cursor: pointer" onclick="window.location='./'">
-            <span style="position: absolute"><img src="images/VA_icon_small.png" style="width: 80px" alt="VA logo and Seal, U.S. Department of Veterans Affairs" /></span>
+            <span style="position: absolute"><img src="images/VA_icon_small.png" style="width: 80px" alt="VA seal, U.S. Department of Veterans Affairs" /></span>
```

Comment:
Just curious if this can or should be an environment variable. Seems to be in a lot of different places. If it were a global it could have been changed once and done.

Reply:
Meant to reply to this.  If they are finally ok with the alt text it might not be a bad idea.